app/controllers/corporate_information_page_controller.rb

```diff
@@ -1,5 +1,6 @@
 class CorporateInformationPageController < ContentItemsController
   def show
     @presenter = CorporateInformationPagePresenter.new(@content_item)
+    @contents_outline_presenter = ContentsOutlinePresenter.new(@presenter.contents_list_headings)
```

Comment:
I think the preferred way of getting at a ContentsOutlinePresenter is to include it in the main presenter behind a method (as here: https://github.com/alphagov/frontend/blob/main/app/presenters/specialist_document_presenter.rb#L5-L9 ) - I appreciate I originally did have separate presenters when I first altered specialist documents, so this is on me.

Reply:
No worries @KludgeKML , that makes sense to include in the main presenter! I've updated in https://github.com/alphagov/frontend/pull/4608/commits/c246acece6b021b63a0f5511ca5fa0ddccb989a3 - `contents_list_headings` is now a private method which is called from a new method`headers_for_contents_list_component` similar to that in specialist documents.